paasta_tools/cli/cmds/status.py

```diff
@@ -135,6 +138,19 @@ def add_subparser(
         default=DEFAULT_SOA_DIR,
         help="define a different soa config directory",
     )
+    status_parser.add_argument(
+        "-k",
+        "--kubernetes",
+        dest="kubernetes",
```

Comment:
should this be a slightly more descriptive flag?
```suggestion
        "--force-kubernetes-status-writer",
        dest="force_kubernetes_status_writer",
```

`--kubernetes` seems a bit too generic imo

Reply:
This flag is remove. As you suggested, I changed the implementation to print all available statuses.